### ISSUE TITLE
Make Game Screen a Better Interface

### ISSUE THREAD

**Issue by Awesomeplayer165** [barryb5/8033_frc_scouting_2022#1]

**Comment by Awesomeplayer165** [comment]
like:
- Remove borders around game screen
- Test interface as with a real game design
Some things to keep in mind:
- has to work without looking at screen or at least minimal screen interaction thats looking down at screen

**Comment by barryb5** [comment]
yeah i agree, one of the things that made your app good was it didn't require looking at the screen to collect data

**Comment by Awesomeplayer165** [comment]
Yeah. I'll leave screenshots of the previous app for Chezy 2022. One of the things we can't take from that is the position of the robot. I'll play around with things to make it still Scouter-friendly.

**Comment by Awesomeplayer165** [comment]
![image](https://user-images.githubusercontent.com/70717139/198093896-7ab12b50-26b6-48f4-bd36-2be68d85a4a7.png)
